### ISSUE TITLE
Mod Dependencies

### ISSUE THREAD

**Issue by m0istn00dl** [DarkPacks/SkyFactory-4#1669]
## Bug Description

When loading, it stops midway through and says that it is missing mod dependencies. It always does this, and I have tried to reinstall the modpack but the problem persists. (This bug also applies to Galacticraft, and I have not been able to successfully run any modpacks with ATLauncher)

### Did this behavior use to work in the previous version?

I don't know, as I have not used previous versions of ATLauncher.

## Steps to Reproduce (for bugs)

1. Download/Reinstall Modpack
2. Run Modpack
3. Repeat as many times as you care to

## Logs

* Client/Server Log: N/A
* Crash Log: N/A

## World Information
* World Preset: N/A
* Prestige Enabled: N/A
* Modpack Version world created in: (Any)
* Additional Content Installed: N/A

## Client Information
* Modpack Version: Latest
* Java Version: The version ATL redirects to
* Launcher Used: ATLauncher
* Memory Allocated: (default)
* Server/LAN/Single Player: N/A
* Optifine Installed: N/A
* Shaders Enabled: N/A

**Comment by sam-kirby** [comment]
I have nothing to add since the last time I responded to this report from you. Spamming us with duplicate reports does not help.

https://github.com/DarkPacks/SkyFactory-4/issues/1665#issuecomment-699703308

tl;dr contact ATLauncher, or use a different launcher. This is not our issue.
